aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -75,11 +75,11 @@ The tutorial assumes familiarity with VS Code. For more information, see [Gettin
  
 [!INCLUDE[](~/includes/run-the-app6.0.md)]
 
-The logs show the service listening on `https://localhost:7042`.
+The logs show the service listening on `https://localhost:<port>`, where `<port>` is the localhost port number randomly assigned when the project is created and set in the *Properties\launchSettings.json.
```

Comment:
```suggestion
The logs show the service listening on `https://localhost:<port>`, where `<port>` is the localhost port number randomly assigned when the project is created and set in `Properties\launchSettings.json`.
```

Reply:
Thanks Kirk!  I'm a little embarrassed I made that mistake.  Thanks for catching it!

Reply:
Ha, no worries. I'm primed to spot these after the monster update I did in this space recently.